E2E/specs/example.spec.js

```diff
@@ -0,0 +1,18 @@
+const env = "local";
+let url = "";
+if (env === "local") {
+  url = "http://127.0.0.1:9000/#/";
+}
+
+describe("opisu zestawu assercji", () => {
+  it("should open website", () => {
+    browser.get(url);
+    var header = element(
+      by.css(
+        "body > todo-app > todo-list > section > todo-header > header > h1"
+      )
+    );
+    var headerText = header.getText();
```

Comment:
var is not necessary here, actually const will be better, var is outdated for this usecase